pkg/grpc/add_metadata_interceptor_test.go

```diff
@@ -0,0 +1,75 @@
+package grpc_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/buildbarn/bb-storage/internal/mock"
+	bb_grpc "github.com/buildbarn/bb-storage/pkg/grpc"
+	"github.com/golang/mock/gomock"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
```

Comment:
Nit: Please add an empty line between imports from separate second level domains (`github.com` vs `golang.org`). This is more consistent with the rest of the codebase.

Reply:
done